articles/postgresql/concepts-firewall-rules.md

```diff
@@ -11,7 +11,7 @@ ms.topic: article
 ms.date: 05/10/2017
 ---
 # Azure Database for PostgreSQL Server firewall rules
-Firewalls prevent all access to your database server until you specify which computers have permission. The firewall grants access to the server based on the originating IP address of each request.
+Azure Database for PostgreSQL Server firewall prevents all access to your database server until you specify which computers have permission. The firewall grants access to the server based on the originating IP address of each request.
```

Comment:
Wanted to make sure we specify that it is the "Azure Database for PostgreSQL Server firewall" that prevents access to the database server. Otherwise, it sounds like it is firewalls in general that prevent access to a database server.